frontend/src/Components/Modals/ModalModifyGame.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+import { Modal, ModalHeader, ModalBody, Button } from 'shards-react';
+
+const ModalModifyGame = ({ open, setOpen, title, action, id, handleGridUpdate, handleOverwriteClick, setUserGamesUpdated }) => {

```

Comment:
I think for simplicity sakes, there should be 3 different modals for Load, Overwrite and Delete, but that can be done in a future PR.